lib/ex_aws/s3/impl.ex

```diff
@@ -351,16 +351,14 @@ defmodule ExAws.S3.Impl do
     |> Map.get(:destination_encryption, %{})
     |> build_encryption_headers
 
-    meta = opts
-    |> Map.get(:meta, %{})
-    |> build_meta_headers
+    regular_headers = opts
+    |> Map.delete(:encryption)
```

Comment:
It's not strictly necessary but copy uses `:source_encryption` and `: destination_encryption` instead.